tests/test_filings.py

```diff
@@ -108,6 +108,21 @@ def test_sort_bad_column(self):
         response = self.app.get(api.url_for(FilingsList, sort='request_type'))
         self.assertEqual(response.status_code, 422)
 
+    def test_sort_multi_column(self):
+        [
+            factories.FilingsFactory(beginning_image_number=2, coverage_end_date='2017-01-01'),
+            factories.FilingsFactory(beginning_image_number=1, coverage_end_date='2017-01-01'),
+            factories.FilingsFactory(beginning_image_number=0, coverage_end_date='2017-01-02'),
+        ]
+
+        results = self._results(api.url_for(FilingsList, sort=['coverage_end_date', 'beginning_image_number']))
+        self.assertEqual(results[0]['beginning_image_number'], '1')
+        self.assertEqual(results[2]['beginning_image_number'], '0')
+        results = self._results(api.url_for(FilingsList, sort=['coverage_end_date', '-beginning_image_number']))
+        self.assertEqual(results[0]['beginning_image_number'], '2')
+        results = self._results(api.url_for(FilingsList, sort=['-coverage_end_date', '-beginning_image_number']))
+        self.assertEqual(results[0]['beginning_image_number'], '0')
```

Comment:
Ok, thanks. I'm on vacation now, so won't be submitting a commit. But my suggestion is to break this up into three tests with descriptive names containing what you said. E.g., 

`def test_old_regular_sort_still_trumps_secondary_sort`

I.e., test names which document the expected outcome.